dmcontent/content_loader.py

```diff
@@ -94,14 +94,14 @@ def get_next_section_id(self, section_id=None, only_editable=False):
     def get_next_editable_section_id(self, section_id=None):
         return self.get_next_section_id(section_id, True)
 
-    def filter(self, context):
+    def filter(self, context, static=False):
```

Comment:
Yeah, good call. I went with static since the only time it's used in a condition is `if static`, but I get how dynamic could be more clear. Changed to `dynamic=False`